commands/experience.py

```diff
@@ -282,7 +282,7 @@ def message_listener(self, event):
                 if len(event.message.mentions) != 1:
                     return
                 for uid in event.message.mentions.keys():
-                    self.handle_action(uid, "submit", False)
+                    self.handle_action(uid, "submit", True)
```

Comment:
this shouldn't be changed, it would impose a limit on how many bugs you can get xp for per day